docs/application/dotnet/guides/nui/window.md

```diff
@@ -313,6 +313,33 @@ child.Unparent();
 > If parent's window stack is changed by calling `Raise()` or `Lower()`, the child window is located on top of the parent again.
 >
 
+### Window Type
```

Comment:
```suggestion
### Window type
```

Reply:
Thank you for detail review.
I will change that.